bin/update-ips.bash

```diff
@@ -522,7 +534,9 @@ if [[ "${check_cluster}" =~ ^(sc|both)$ ]]; then
   allow_nodes "sc" '.networkPolicies.global.scApiserver.ips' "node-role.kubernetes.io/control-plane="
   allow_nodes "sc" '.networkPolicies.global.scNodes.ips' ""
 
-  allow_swift
+  if swift_enabled; then
+    sync_swift '.objectStorage.swift.authUrl' '.networkPolicies.global.objectStorageSwift'
```

Comment:
```suggestion
    sync_swift '.objectStorage.swift' '.networkPolicies.global.objectStorageSwift'
```
I think you need to do something like this so you can pass it on to `get_swift_url`, else you will not run with the correct credentials for `.objectStorage.sync.swift` below.